src/python/detectors/integer_overflow/integer_overflow.py

```diff
@@ -0,0 +1,20 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=integer-overflow@v1.0 defects=1}
+def integer_overflow_noncompliant():
+    import numpy as np
```

Comment:
Since np is used by all methods here, can we move it up to, say, line 3?

Reply:
Moved the import .